README.md

```diff
@@ -4,19 +4,22 @@
 [![Coverage Status](https://coveralls.io/repos/github/observatorycontrolsystem/ingester/badge.svg?branch=master)](https://coveralls.io/github/observatorycontrolsystem/ingester?branch=master)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/24eb8debeb0c499ca192b4497a1f1e12)](https://www.codacy.com/gh/observatorycontrolsystem/ingester?utm_source=github.com&utm_medium=referral&utm_content=observatorycontrolsystem/ingester&utm_campaign=Badge_Grade)
 
-A library for adding new science data products to an observatory control system's science archive. The library
-handles uploading files into AWS S3, as well as adding records to the science archive's database containing
-the searchable metadata of all available files. It records metrics in an openTSDB about ingestion.
+A library for adding new science data products to an observatory control system's [Science Archive](https://github.com/observatorycontrolsystem/science-archive/).
+The library handles uploading files into a `FileStore` (usually S3), as well as adding records to the Science Archive's database containing
+the searchable metadata of all available files. It records ingestion metrics in an openTSDB instance, which can be disabled by setting the environment variable `OPENTSDB_PYTHON_METRICS_TEST_MODE=False`.
```

Comment:
**Optionally** it records ingestion metrics in an [OpenTSDB](http://opentsdb.net/) (add a link) instance.